homeassistant/components/assist_satellite/entity.py

```diff
@@ -57,25 +57,31 @@ async def _async_accept_pipeline_from_satellite(
         pipeline_id: str | None = None
         vad_sensitivity = vad.VadSensitivity.DEFAULT
 
-        if pipeline_entity_id:
+        if (
+            pipeline_entity_id
+            and (
+                (pipeline_entity_state := self.hass.states.get(pipeline_entity_id))
```

Comment:
I think that we should raise if the pipeline entity does not exist. It means the caller passed an invalid value, which we shouldn't silently ignore.